### ISSUE TITLE
Syncing untranslated strings makes them become blank text

### ISSUE THREAD

**Issue by jkobus** [php-translation/loco-adapter#14]
1. I have 8 untranslated headings
2. I press "Sync selected translations to SaaS"
3. My translation file is filled with: translation.key: ""
4. Untranslated strings disappear on the website

I would rather expect these strings to be, by default, the same as the key.
Also, the fallback language will not work in this case.

What is the expected scenario here?

**Comment by Nyholm** [comment]
If you sync them and there is nothing in Loco, it should be no translation locally. An empty string is not good enough. An empty string would represent "translation found, it is an empty string". 

I recently made [a fix](https://github.com/php-translation/loco-adapter/commit/548e57ade4f4ab9766f9fd3a5d41f8170a9524f9) to only get translated strings from loco. Maybe the same fix should be applied to the sync command?

**Comment by jkobus** [comment]
To be more precise, the problem is that the bundle sends the translations to local file storage. 
Default translation is set to '' (blank string). There is no way to recognize if this was intentional empty string or translation was not provided. By default symfony sets default translation to the same value as the key:

button.show_more = "button_show_more"

In SymfonyProfilerController there is a comment:

> //We do not want do add the placeholder to Loco. That messes up the stats.

I belive that is the reason why we intentionally set the new translation as a blank string. But this does not work in case of xliff files.

I belive that the translation should take place on the loco, so translation can be downloaded when its done by the translators. Until then, it should be displayed as a "key". 

We should also be able to add new translation to our "temporary" translation files during development. 
Therefore I think it will be wise to set default translation string to the same value as translation key but only for FileStorage. The only problem is that the "sync" command will send these key-to-key translations to Loco and that will indeed, mess up the stats.

The support for null translation might be also helpful, if we would like to add an option to support "intentionally blank" option on Loco.

**Comment by jkobus** [comment]
Another option is to ignore translations that are the same as keys in loco-adapter:

Loco.php:
```
// translation is the same as the key, so we will set it to empty string
        // as it was not translated and stats on loco will be unaffected
        if($message->getKey() === $message->getTranslation()){
            $translation = '';
        }
```

We would be able to use default symfony's placeholders for translation but ignore it only for loco.
There might be a problem when the key will be the same as the real translation, tough. For example:

`"ok" = "ok"`

PR https://github.com/php-translation/loco-adapter/pull/16

**Comment by Nyholm** [comment]
Closed by #16
